### PR TITLE
Rspec setup

### PR DESCRIPTION
**Pull request by HannaSwDn**
## Pivotal Tracker link
[Link to PT](https://www.pivotaltracker.com/story/show/162379504)

## Actions taken
Creates Rspec testing environment
Adds HAML to project

## What we have learned
How to setup Rspec and resolve Gemfile.lock merge conflict

### CODE REVIEW

**Review by shahin1290 — APPROVED** [review]

**Review by schneiderman805 — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -8,12 +8,19 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
```
Comment (faraznaeem) [inline Gemfile]: why have you readded coffee-rails?

File: Gemfile [hunk 2]
```diff
@@ -8,12 +8,19 @@ gem 'pg', '>= 0.18', '< 2.0'
...
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
-
 gem 'bootsnap', '>= 1.1.0', require: false
+gem 'haml-rails'
```
Comment (faraznaeem) [inline Gemfile:14]: is haml in the description

File: Gemfile [hunk 3]
```diff
@@ -25,4 +32,4 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
-
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (faraznaeem) [inline Gemfile]: please remove this

File: config/application.rb [hunk 4]
```diff
@@ -11,23 +10,19 @@
...
+      generate.assets false
+      generate.view_specs false
+      generate.helper_specs false
+      generate.routing_specs false
+      generate.controller_specs false
+      generate.system_tests false
+    end
   end
-end
+end
```
Comment (faraznaeem) [inline config/application.rb:28]: Clean this file up from comments

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Comment by faraznaeem** [comment]
Reviews from team members need to be more thorough

File: Gemfile (hunk 2 above)
Comment (faraznaeem) [reply Gemfile:14]: if not then remove it

File: Gemfile (hunk 1 above)
Comment (HannaSwDn) [reply Gemfile]: We got merge conflicts and accepted both changes, but removing now

File: Gemfile (hunk 2 above)
Comment (HannaSwDn) [reply Gemfile:14]: Added to description

File: Gemfile (hunk 3 above)
Comment (HannaSwDn) [reply Gemfile]: Removed

**Review by shahin1290 — APPROVED** [review]

File: config/application.rb (hunk 4 above)
Comment (HannaSwDn) [reply config/application.rb:28]: Removed

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**